resources/aix/pxp-agent.service

```diff
@@ -1 +1 @@
-'/opt/puppetlabs/puppet/bin/pxp-agent' -a '--foreground' -s pxp-agent -u root
+/opt/puppetlabs/bin/pxp-agent -a '--foreground' -s pxp-agent -u root
```

Comment:
Not blocking, just curious if `'--foreground'` needs to be quoted here.

Reply:
Looking at https://github.com/puppetlabs/puppet-agent/blob/main/resources/aix/puppet.service#L1, I think `-a` means "arguments" so it probably should be.